modules/azure/compute.go

```diff
@@ -0,0 +1,102 @@
+package azure
+
+import (
+	"context"
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2018-06-01/compute"
+	"github.com/stretchr/testify/require"
+)
+
+// getVirtualMachineClient is a helper function that will setup an Azure Virtual Machine client on your behalf
+func getVirtualMachineClient(subscriptionID string) (compute.VirtualMachinesClient, error) {
```

Comment:
Should this be a public method?

Reply:
It wasn't intended to be, but I could see the merit of making it one for the sake of letting users consume the client it creates.  Will change it up.